cfme/configure/tasks.py

```diff
@@ -40,7 +40,9 @@
     ]
 )
 
-table_loc = '//div[@id="records_div"]/table'
+table_loc = VersionPick({
+    '5.9': '//div[@class="miq-data-table"]/table',
```

Comment:
**Optional**: Can use only one selector for all versions '//div[@id="gtl_div"]//table', and omit VersionPick

Reply:
I would make it **REQUIRED**

Reply:
this also springs in my mind when I see this statement

Reply:
Thanks guys/girls, good find, good to know that covers all z-streams